extra/mender-device-docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+
+/etc/init.d/ssh start
+cp /usr/share/dbus-1/system.d/io.mender.AuthenticationManager.conf /etc/dbus-1/system-local.conf
```

Comment:
I don't know why we need this. I took it from https://github.com/mendersoftware/gui/blob/master/tests/e2e_tests/dockerClient/clientEntrypoint.sh#L4

Reply:
dbus didn't start without the file being present at that location... 🤷‍♂️ - I guess @kacf knows a lot more about dbus 😅

Reply:
that might depend on the base image `mendersoftware/mender-client-docker:master` uses - but the image building for the client images is opaque to me...

Reply:
I'll leave this workaround (at least for now) unless Kristian knows the right fix.